sql/core/src/main/java/org/apache/spark/sql/sources/v2/reader/DataReaderFactory.java

```diff
@@ -50,7 +50,7 @@
   }
 
   /**
-   * Returns a data reader to do the actual reading work for this read task.
+   * Returns a data reader to do the actual reading work for this data reader factory.
```

Comment:
we can follow the `DataWriterFactory` and just say `... to do the actual reading work`